src/utils/options.js

```diff
@@ -129,6 +130,23 @@ const baseSchema = {
                     "description": "Min and max values"
                 }
             }
+        },
+        "zoomInfo": {
+            "type": "object",
+            "properties": {
```

Comment:
```suggestion
            "required": ["level", "top", "numRows"],
            "properties": {
```

Reply:
Just a minor thing, but can we expect that these keys will be provided every time?